ci/etcd.rb

```diff
@@ -0,0 +1,51 @@
+require './ci/common'
+
+namespace :ci do
+  namespace :etcd do |flavor|
+    task :before_install => ['ci:common:before_install']
+
+    task :install => ['ci:common:install'] do
+      sh %(curl -s -L -o $VOLATILE_DIR/etcd.tar.gz\
+            https://github.com/coreos/etcd/releases/download/v2.0.3/etcd-v2.0.3-linux-amd64.tar.gz)
+      sh %(mkdir $VOLATILE_DIR/etcd)
+      sh %(tar xzvf $VOLATILE_DIR/etcd.tar.gz\
+                    -C $VOLATILE_DIR/etcd\
+                    --strip-components=1 >/dev/null)
+    end
+
+    task :before_script => ['ci:common:before_script'] do
+      sh %(cd $VOLATILE_DIR/etcd && ./etcd >/dev/null &)
```

Comment:
You should install the binaries and integration in `embedded` and not `VOLATILE_DIR`, the idea is that `embedded` is cached and we don't reinstall the integration if it's already here. `VOLATILE_DIR` on the contrary is disposable (and useful for instance as a build area).